TimePlatformer/Assets/Scripts/FireScript.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class FireScript : MonoBehaviour {
+	
+
+	// Use this for initialization
+	void Start () {
+
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	
+	}
+
+	void OnCollisionEnter2D (Collision2D col){
+		col.gameObject.transform.position = new Vector3 (0f, 1.5f, 0f);
```

Comment:
Can we change this into a public transform object, or a vector3 so then we can change this via the Unity Editor as opposed to changing code?